src/react/features/lenses/timeline/hooks/getTimelineData.js

```diff
@@ -4,10 +4,10 @@ const getAverage = (array = []) =>
 const concat = (array = [], item) => (typeof item !== 'undefined' ? array.concat(item) : array)
 
 export const getDateLongitude = ({ date }) =>
-  (Date.parse(date) / Date.parse('1971-01-01T00:00:00.000Z')) * 10
+  date ? (Date.parse(date) / Date.parse('1971-01-01T00:00:00.000Z')) * 10 : 0
```

Comment:
I'm afraid this cannot be like this and will cause many problems in timeline. At least it cannot be zero but undefined or null.